### PR TITLE
Flash powder now stuns cyborgs too

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request
Flash powder stuns cyborgs like flashes and flashbangs do

## Why It's Good For The Game
Consistency and support for whoever eventually makes flashbangs work like sonic powder/flash powder grenades

## Changelog
:cl:
tweak: flash powder stuns borgs
/:cl:

### CODE REVIEW

**Comment by PKPenguin321** [comment]
REEEEEEE FIX https://github.com/tgstation/tgstation/issues/40144 INSTEAD OF FURTHER SNOWFLAKING THE COPYPASTED CODE

**Comment by carlarctg** [comment]
yikes effort? no thanks

File: code/modules/reagents/chemistry/recipes/pyrotechnics.dm [hunk 1]
```diff
@@ -245,8 +245,8 @@
 	if(isatom(holder.my_atom))
 		var/atom/A = holder.my_atom
 		A.flash_lighting_fx(_range = (range + 2), _reset_lighting = FALSE)
-	for(var/mob/living/carbon/C in get_hearers_in_view(range, location))
-		if(C.flash_act())
+	for(var/mob/living/C in get_hearers_in_view(range, location))
+		if(C.flash_act(affect_silicon = 1))
```
Comment (81Denton) [inline code/modules/reagents/chemistry/recipes/pyrotechnics.dm]: TRUE

File: code/modules/reagents/chemistry/recipes/pyrotechnics.dm [hunk 2]
```diff
@@ -266,8 +266,8 @@
...
```
Comment (81Denton) [inline code/modules/reagents/chemistry/recipes/pyrotechnics.dm]: TRUE

**Review by 81Denton — CHANGES_REQUESTED** [review]

File: code/modules/reagents/chemistry/recipes/pyrotechnics.dm (hunk 2 above)
Comment (carlarctg) [reply code/modules/reagents/chemistry/recipes/pyrotechnics.dm]: done

**Pull request merged**